server/webserver.go

```diff
@@ -543,6 +543,37 @@ func (s *LivepeerServer) StartWebserver() {
 				return
 			}
 
+			dAddr := s.LivepeerNode.Eth.Account().Address
+
+			d, err := s.LivepeerNode.Eth.GetDelegator(dAddr)
+			if err != nil {
+				glog.Error(err)
+				return
+			}
+			latestUnbondingLockID, err := s.LivepeerNode.Database.LatestUnbondingLockID()
+			if err != nil {
+				glog.Error(err)
+				return
+			}
+
+			// Update unbonding locks in local DB if necessary
```

Comment:
I wonder if we could (also) do this fixup during node startup to help CLI responsiveness a bit. But that's something of a premature optimization.

Reply:
I think we could hold off on this for now given that these fix ups are interim solutions that could potentially be discarded in a future update if we can just rely on events